public/app/shared/activityTimelineEventDetails.html

```diff
@@ -31,28 +31,25 @@ <h3 class="event-details-title" data-ng-if="pageContext.tool === 'dashboard'">
     </h3>
 
     <p class="event-details-description">
-      <span data-ng-if="pageContext.tool === 'dashboard'">
-        <i class="fa fa-comment" data-ng-if="display.comment"></i>
-        <i class="fa fa-thumbs-up" data-ng-if="display.like"></i>
-        <i class="fa fa-eye" data-ng-if="display.view"></i>
-        <strong data-ng-bind="display.description"></strong>
-      </span>
-
-      <span data-ng-if="pageContext.tool === 'assetlibrary'">
-        <a tool-href
-           data-tool="dashboard"
-           data-id="activity.user.id"
-           data-referring-tool="pageContext.tool"
-           data-referring-id="pageContext.id"
-           class="event-details-strong"
-           data-ng-bind="activity.user.canvas_full_name"></a>
-        <span data-ng-switch="activity.type">
-          <span data-ng-switch-when="view_asset">viewed this asset.</span>
-          <span data-ng-switch-when="like">liked this asset.</span>
-          <span data-ng-switch-when="asset_comment">commented on this asset, </span>
-          <span data-ng-switch-when="whiteboard_add_asset">used this asset in a whiteboard.</span>
-          <span data-ng-switch-when="remix_whiteboard">remixed this asset in a whiteboard.</span>
-        </span>
+      <a tool-href
+         data-tool="dashboard"
+         data-id="activity.user.id"
+         data-referring-tool="pageContext.tool"
+         data-referring-id="pageContext.id"
+         class="event-details-strong"
+         data-ng-bind="activity.user.canvas_full_name"></a>
+      <span data-ng-switch="activity.type">
+        <span data-ng-switch-when="add_asset">added this asset.</span>
+        <span data-ng-switch-when="view_asset">viewed this asset.</span>
+        <span data-ng-switch-when="get_view_asset">viewed this asset.</span>
+        <span data-ng-switch-when="like">liked this asset.</span>
+        <span data-ng-switch-when="get_like">liked this asset.</span>
+        <span data-ng-switch-when="asset_comment">commented on this asset, </span>
+        <span data-ng-switch-when="get_asset_comment">commented on this asset, </span>
+        <span data-ng-switch-when="whiteboard_add_asset">used this asset in a whiteboard.</span>
+        <span data-ng-switch-when="get_whiteboard_add_asset">used this asset in a whiteboard.</span>
+        <span data-ng-switch-when="remix_whiteboard">remixed this asset in a whiteboard.</span>
+        <span data-ng-switch-when="get_remix_whiteboard">remixed this asset in a whiteboard.</span>
```

Comment:
@johncrossman Good catch on pins! I'll add it. On `data-ng-if="pageContext.tool === 'assetlibrary'"`, I _think_ we don't need that because the new link is also replacing the `pageContext.tool === 'dashboard'"` link below. (That is, we're now consistent about where user name sits.) In either case, we're linking to the dashboard from pageContext.tool.